client/src/Categories/Categories.js

```diff
@@ -12,6 +13,9 @@ function Categories() {
       <Route exact path={path}>
         <CategoriesList />
       </Route>
+      <Route path={`${path}/:slug/resources`}>
```

Comment:
Let's just make the path `${path/:slug}` without the resources appended to it. That way, this becomes more of a generic landing page for a category- it may show resources, it may also (with more development!) show links to questionnaires, etc...

Maybe then the component just becomes `Category` ...?

Reply:
I have fixed this issue and removed `/resources`